.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "standard-with-typescript",
```

Comment:
I recommend use "airbnb-base" extend. It will contain everything you need, and there is also for typescript.